src/mono/wasm/runtime/exports.ts

```diff
@@ -376,7 +387,12 @@ export const __linker_exports: any = {
     // pal_crypto_webworker.c
     dotnet_browser_can_use_subtle_crypto_impl,
     dotnet_browser_simple_digest_hash,
-    dotnet_browser_sign
+    dotnet_browser_sign,
+
+    // mono-threads-wasm.c
+    ...(MonoWasmThreads) && {
```

Comment:
are `...` also unpacking the object after `&&` ?

Reply:
StackOverflow told me this is how you make ... either unpack the `{ }` object after the `&&` or expand to nothing if `MonoWasmThreads` is falsy.

Reply:
https://jsfiddle.net/65hLcotp/6/

![Screen Shot 2022-06-22 at 10 23 39](https://user-images.githubusercontent.com/480437/175053463-7cb565ad-dec8-4872-937f-14437da8229d.png)
![Screen Shot 2022-06-22 at 10 23 50](https://user-images.githubusercontent.com/480437/175053465-34d8b35e-d660-4e96-a416-10373a169438.png)

Reply:
Okay, more readable would be. 

```
let x = {a:"a",b:"b",c:"c"} ;
if (flag){
    x = {
        ...x, 
        d:"d"
    };
}

```

No so cool tho ;)

Reply:
but then it's not `const` :cry: 

I'll clean it up.